slides/shared/about-slides.md

```diff
@@ -6,12 +6,12 @@
 
 - You can get updated "builds" of the slides there:
 
-  http://container.training/
+  @@SLIDES@@
 
 <!--
 .exercise[
 ```open https://@@GITREPO@@```
-```open http://container.training/```
+```open @@SLIDES@@```
```

Comment:
This is in a commented section, used only by the autopilot. It's using container.training on purpose, because the public link may or may not be up while running the autopilot.